static/js/controllers/user-language-modal.js

```diff
@@ -0,0 +1,64 @@
+  var inboxServices = angular.module('inboxServices');
+
+  inboxServices.factory('UserLanguageModal', ['$uibModal',
+    function($uibModal) {
+      return function(enabledLocales) {
+        var modalInstance = $uibModal.open({
+          templateUrl: 'templates/modals/user_language.html',
+          controller: 'UserLanguageModalCtrl',
+          resolve: {
+            enabledLocales: function () {
+              return enabledLocales;
+            }
+          }
+        });
+        return modalInstance.result;
+      };
+    }
+  ]);
+
+angular.module('inboxControllers').controller('UserLanguageModalCtrl',
+  ['enabledLocales', '$log', '$q', '$scope', 'Session', 'SetLanguage', '$translate', '$uibModalInstance', 'UpdateUser',
```

Comment:
I can't find anything about 'enabledLocales' either in develop or this PR. I assume it does what it says on the box. Can you rename it to "EnabledLocales" as per the convention?